dev/com.ibm.ws.webcontainer.servlet.3.1_fat/fat/src/com/ibm/ws/webcontainer/servlet31/fat/tests/CDIBeanInterceptorServletTest.java

```diff
@@ -188,7 +181,7 @@ public String getParameterText(String... parms) {
      * @return The request url.
      */
     public String getCurrencyURL(String... parms) {
-        return CURRENCY_INTERCEPTOR_URL + getParameterText(parms);
+        return CURRENCY_INTERCEPTOR_URL_FRAGMENT + getParameterText(parms);
```

Comment:
Why this change?

Reply:
For the URL changes the URL variable was a combination of `BLANK_CONTEXT_ROOT + BLANK_URL_FRAGMENT` and because the private `verifyStringsInResponse` function already takes in the `contextRoot` and remaining `path` we didn't need the combined URL variable anymore.